src/styles/_catalog.scss

```
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

```suggestion
  grid-template-columns: 200px;
```
move `200px` into variables